python/lsst/ip/diffim/utils.py

```diff
@@ -887,11 +880,13 @@ def getPsfFwhm(psf, average=True, position=None):
     """
     if position is None:
         position = psf.getAveragePosition()
-    image = psf.computeKernelImage(position).array
-    peak = psf.computePeak(position)
-    peakLocs = np.unravel_index(np.argmax(image), image.shape)
-    width = _sliceWidth(image, peak/2., peakLocs, axis=0), _sliceWidth(image, peak/2., peakLocs, axis=1)
-    return np.nanmean(width) if average else width
+    shape = psf.computeShape(position)
+    sigmaToFwhm = 2*np.log(2*np.sqrt(2))
+
+    if average:
+        return sigmaToFwhm*shape.getTraceRadius()
+    else:
+        return [sigmaToFwhm*np.sqrt(shape.getIxx()), sigmaToFwhm*np.sqrt(shape.getIyy())]
```

Comment:
Wasn't this merged before? I lost track of this, thought that it was on your previous ticket: DM-43641

Reply:
Sorry. I rebased the branch with main, and now this is not longer a change in this PR.